tests/functional/filter/cram/filter-metadata-sequence-strains-mismatch.t

```diff
@@ -10,20 +9,25 @@ The list of strains to include has one strain with no metadata/sequence and one
 The query initially filters 3 strains from Colombia, one of which is added back by the include.
 
   $ ${AUGUR} filter \
-  >  --sequence-index filter/data/sequence_index.tsv \
-  >  --metadata filter/data/metadata.tsv \
+  >  --sequence-index "$TESTDIR/../data/sequence_index.tsv" \
+  >  --metadata "$TESTDIR/../data/metadata.tsv" \
   >  --query "country != 'Colombia'" \
   >  --non-nucleotide \
   >  --exclude-ambiguous-dates-by year \
-  >  --include filter/data/include.txt \
-  >  --output-strains "$TMP/filtered_strains.txt" \
-  >  --output-log "$TMP/filtered_log.tsv"
+  >  --include "$TESTDIR/../data/include.txt" \
+  >  --output-strains filtered_strains.txt \
+  >  --output-log filtered_log.tsv
   4 strains were dropped during filtering
   \t1 had no metadata (esc)
   \t1 had no sequence data (esc)
   \t3 of these were filtered out by the query: "country != 'Colombia'" (esc)
-  \t1 strains were added back because they were in filter/data/include.txt (esc)
+  \\t1 strains were added back because they were in .*include\.txt.* (re)
   9 strains passed all filters
 
-  $ diff -u <(sort -k 1,1 filter/data/filtered_log.tsv) <(sort -k 1,1 "$TMP/filtered_log.tsv")
-  $ rm -f "$TMP/filtered_strains.txt"
+  $ sort -k 1,1 filtered_log.tsv
+  COL/FLR_00008/2015\tfilter_by_query\t"[[""query"", ""country != 'Colombia'""]]" (esc)
+  COL\/FLR_00008\/2015\\tforce_include_strains\\t"\[\[""include_file"", "".*""\]\]".* (re)
+  COL/FLR_00024/2015\tfilter_by_query\t"[[""query"", ""country != 'Colombia'""]]" (esc)
+  Colombia/2016/ZC204Se\tfilter_by_query\t"[[""query"", ""country != 'Colombia'""]]" (esc)
+  HND/2016/HU_ME59\tfilter_by_sequence_index\t[] (esc)
+  strain\tfilter\tkwargs (esc)
```

Comment:
_non-blocking_

To make this a bit more readable, how about using literal tabs in the expected output and `(glob)` instead of `(re)`? (I also omitted the header from the sort to reduce a brief moment of wut.)

```
  $ (head -n1; sort -k 1,1) < filtered_log.tsv
  strain	filter	kwargs
  COL/FLR_00008/2015	filter_by_query	"[[""query"", ""country != 'Colombia'""]]"
  COL/FLR_00008/2015\tforce_include_strains\t"[[""include_file"", ""*/data/include.txt""]]" (esc) (glob)
  COL/FLR_00024/2015	filter_by_query	"[[""query"", ""country != 'Colombia'""]]"
  Colombia/2016/ZC204Se	filter_by_query	"[[""query"", ""country != 'Colombia'""]]"
  HND/2016/HU_ME59	filter_by_sequence_index	[]
```

Unfortunately, for the one variable line, cram considers tabs to be "unprintable" and thus expects to match against the `(esc)`-d form which `(glob)` has to match against.

Reply:
That looks a lot better! Updated in [force-push](https://github.com/nextstrain/augur/compare/7f738e5c63f6a7d3ebf8e871b944a87f72cc7baf..5d2e370907dc1174cb1efc04292059ca63934cd4).